### PR TITLE
Bucky/docs clean

### PR DESCRIPTION
**Pull request by ebuchman**
Update repo docs to facilitate getting started quicker.

Add a security.md to describe bug bounty.

### CODE REVIEW

**Comment by ebuchman** [comment]
@jessysaurusrex see update to README.md and the new SECURITY.md

@zramsay could use help with the networks/remote/README.md

File: Makefile [hunk 1]
```diff
@@ -193,6 +193,10 @@ build-docker:
 build-linux:
 	GOOS=linux GOARCH=amd64 $(MAKE) build
 
+build-docker-localnode:
```
Comment (melekes) [inline Makefile]: please add to `.PHONY`

File: networks/remote/README.md [hunk 2]
```diff
@@ -0,0 +1 @@
+# Remote Cluster with Terraform and Ansible
```
Comment (zramsay) [inline networks/remote/README.md]: see docs/terraform-and-ansible.rst

Comment (ebuchman) [reply networks/remote/README.md]: ok fixed in https://github.com/tendermint/tendermint/pull/1602 thanks

**Pull request merged**